src/compiler/notification-groups.md

```diff
@@ -0,0 +1,39 @@
+# Notification groups
+
+The compiler team has a number of notification groups that we use to
+ping people and draw their attention to issues. Notification groups
+are setup so that anyone can join them if they want.
+
+## Creating a notification group
+
+If you'd like to create a notification group, here are the steps.
+First, you want to get approval:
+
+* Propose the group by preparing a [Major Change Proposal][MCP].  If
+  your group is not analogous to some existing group, it is probably
+  a good idea to ping compiler team leads before-hand or as part of
+  the MCP.
+* The MCP should specify what GitHub label will be associated with the
+  notification group. Often this is an existing label, such as
+  `O-Windows`.
+  
+Once the MCP is seconded, here are the steps to actually create the group.
```

Comment:
Just to be sure, should this whole procedure of filing PRs begin once the MCP is seconded or accepted?

Reply:
```suggestion
Once the MCP is accepted, here are the steps to actually create the group.
```

good point